lib/main.dart

```diff
@@ -26,6 +28,8 @@ void main(List<String> args) async {
 
   await closeExistingSessions();
 
+  if(platformIsDesktop()) hotKeyService.initHotkeyRegistration();
```

Comment:
This should happen after the logger is initialized, so the service can use the
logger if needed.

Reply:
Noted!